src/ResultPage.js

```diff
@@ -132,6 +134,8 @@ function ResultPage({
                         </MeasureList>
                     </Information>
                 </>
+            ) : (
+                status === 'error' && history.push('/error')
```

Comment:
Are there other possible statuses? If not, you could remove the `` status === 'error' &&``.

Reply:
yes, status has four different states :)